pkg/registry/core/node/storage/storage_test.go

```diff
@@ -54,12 +54,14 @@ func newStorage(t *testing.T) (*REST, *etcd3testing.EtcdTestServer) {
 	return storage.Node, server
 }
 
-func validNewNode() *api.Node {
-	return &api.Node{
+type Tweak func(*api.Node)
```

Comment:
do we need to export it? why is it public?

Reply:
not necessarily, I just amended it to be private now.